elide-core/src/main/java/com/yahoo/elide/core/DataStoreTransaction.java

```diff
@@ -164,12 +165,15 @@ default void preCommit() {
             String relationName,
             Class<T> relationClass,
             EntityDictionary dictionary,
-            Set<Predicate> filters
+            FilterScope filterScope
```

Comment:
getting `MultiplexTransaction` through `filterScope.getRequestScope().getTransaction()` sounds a bit twisted. I was thinking to add a `Transaction getParentTransaction()` method. What do you guys think?